docs/metrics.md

```diff
@@ -49,21 +58,22 @@ object MetricsApp extends ZIOAppDefault {
 
   override def run: ZIO[Scope, Any, ExitCode] =
     (for {
-      _            <- ZIO.logInfo("Start")
-      _            <- ZIO.logWarning("Some warning")
-      _            <- ZIO.logError("Some error")
-      _            <- ZIO.logError("Another error")
-      _            <- ZIO.sleep(1.second)
+      _ <- ZIO.logInfo("Start")
+      _ <- ZIO.logWarning("Some warning")
+      _ <- ZIO.logError("Some error")
+      _ <- ZIO.logError("Another error")
+      _ <- ZIO.sleep(1.second)
       metricValues <- ZIO.serviceWithZIO[PrometheusPublisher](_.get)
-      _            <- Console.printLine(metricValues)
-      _            <- ZIO.logInfo("Done")
+      _ <- Console.printLine(metricValues)
+      _ <- ZIO.logInfo("Done")
     } yield ExitCode.success)
       .provideLayer((ZLayer.succeed(MetricsConfig(200.millis)) ++ publisherLayer) >+> prometheusLayer)
 
 }
 ```
 
 Expected Console Output:
+
```

Comment:
in relation to metric changes, console output for this example probably also changed, 
i think you should run MetricsApp from examples and update it here